README.md

```diff
@@ -133,7 +174,7 @@ AbstractMulticopter vehicle = new Quadcopter(world, "models/3dr_arducopter_quad_
 
 Custom MAVLink protocols can be used, no any recompilation needed, just specify XML file instead of `custom.xml` here:
 ```
-MAVLinkSchema schema = new MAVLinkSchema("mavlink/message_definitions/common.xml");
+MAVLinkSchema schema = new MAVLinkSchema("mavlink/message_definitions/common.xml
```

Comment:
Why did you change this?

Reply:
Sorry, It's my mistake.. I changed it again.